.travis.yml

```diff
@@ -10,20 +10,21 @@ sudo: false
 
 cache:
   directories:
+    - .phpunit
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=2.8.*
-  global: SYMFONY_DEPRECATIONS_HELPER=321
+  matrix: SYMFONY_VERSION=3.2.*
+  global: SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
 
 matrix:
   include:
-    - php: 7.0
-      env: DEPS=dev SYMFONY_VERSION=3.1.*
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION=3.3.*
     - php: 5.5
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
-      env: DEPS=dev COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.*
```

Comment:
Symfony 3.0 has reached end of maintaince, let's stop testing against it.

@symfony-cmf/core as we test against Symfony 2.8 and Symfony 3.1+, we can be very confident that we support 3.0. Shall we keep allowing it in composer.json, or shall we set it to `2.8.*|^3.1`?